src/sentry/static/sentry/app/components/group/externalIssueActions.jsx

```diff
@@ -162,67 +161,80 @@ class ExternalIssueForm extends AsyncComponent {
   }
 }
 
-class ExternalIssueActionList extends AsyncComponent {
+class ExternalIssueActions extends AsyncComponent {
   static propTypes = {
     group: PropTypes.object.isRequired,
+    integration: PropTypes.object.isRequired,
   };
 
   constructor(props, context) {
     super(props, context);
     this.state.showModal = false;
-    this.state.selectedIntegration = null;
-    this.state.action = null;
+    this.state.selectedIntegration = this.props.integration;
+    this.state.action = 'create';
+    this.state.issue = this.getIssue();
   }
 
-  getEndpoints() {
-    let {group} = this.props;
-    return [['integrations', `/groups/${group.id}/integrations/`]];
+  getIssue() {
+    return this.props.integration.externalIssues[0];
```

Comment:
Can we be a bit more defensive here in case `integration` or `externalIssues` doesn't exist?